src/angularjs/package.json

```diff
@@ -4,7 +4,7 @@
   "license": "MIT",
   "scripts": {
     "ng": "ng",
-    "start": "ng serve",
+    "start": "ng serve --host 0.0.0.0",
```

Comment:
If we supply this via `command` in the `docker-compose.yml`, will it work? If so, I think it would be best to add this flag only when we know we're running inside of a container vs. if someone clones the project and runs `npm start` directly on their workstation.